File: iniciante/1050/1050.cpp
// 1050 - DDD - beecrowd

#include <iostream>

using std::cout;

int main(void)
{
    int ddd;
    std::cin >> ddd;

    switch (ddd)
    {
        case 61:
            cout << "Brasilia\n";
            break;
        case 71:
            cout << "Salvador\n";
            break;
        case 11:
            cout << "Sao Paulo\n";
            break;
        case 21:
            cout << "Rio de Janeiro\n";
            break;
        case 32:
            cout << "Juiz de Fora\n";
            break;
        case 19:
            cout << "Campinas\n";
            break; 
        case 27:
            cout << "Vitoria\n";
            break;
        case 31:
            cout << "Belo Horizonte\n";
            break;
        default:
            cout << "DDD nao cadastrado\n";
            break;
    }

    return 0;
}

File: iniciante/1050/1050.js
// 1050 - DDD - beecrowd

var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

const ddd = parseInt(lines[0]);
let cidade;

switch(ddd){
	case 61:
		cidade = 'Brasilia';
		break;
	case 71:
		cidade = 'Salvador';
		break;
	case 11:
		cidade = 'Sao Paulo';
		break;
	case 21:
		cidade = 'Rio de Janeiro';
		break;
	case 32:
		cidade = 'Juiz de Fora';
		break;
	case 19:
		cidade = 'Campinas';
		break;
	case 27:
		cidade = 'Vitoria';
		break;
	case 31:
		cidade = 'Belo Horizonte';
		break;
	default:
		cidade = 'DDD nao cadastrado';
}

console.log(cidade);
